src/NexusMods.App.UI/Pages/ILoadoutDataProvider.cs

```diff
@@ -25,6 +28,15 @@ public static IObservable<IChangeSet<Datom, EntityId>> FilterInStaticLoadout(
         IConnection connection,
         LoadoutFilter loadoutFilter)
     {
+        if (loadoutFilter.CollectionGroupId.HasValue)
+        {
+            return source.Filter(datom =>
+                {
+                    var item = LoadoutItem.Load(connection.Db, datom.E);
+                    return item.LoadoutId == loadoutFilter.LoadoutId && item.IsChildOf(loadoutFilter.CollectionGroupId.Value);
+                }
+            );
+        }
```

Comment:
```suggestion
        var filterByCollection = loadoutFilter.CollectionGroupId.HasValue;
        return source.Filter(datom =>
        {
            var item = LoadoutItem.Load(connection.Db, datom.E);
            if (!item.LoadoutId.Equals(loadoutFilter.LoadoutId)) return false;
            if (filterByCollection) return item.IsChildOf(loadoutFilter.CollectionGroupId.Value);
            return true;
        });
```

A bit more compact and readable.